src/imitation/scripts/config/train.py

```diff
@@ -140,6 +141,30 @@ def swimmer():
   )
 
 
+@train_ex.named_config
+def walker():
+  env_name = "Walker2d-v2"
+
+
+# Custom env configs
+
+@train_ex.named_config
+def two_d_maze():
+  env_name = "imitation/TwoDMaze-v0"
+
+
+@train_ex.named_config
+def custom_ant():
```

Comment:
Why does `ant()` get `n_epochs = 2000` but not `custom_ant` or `disabled_ant`?

I also suspect we may need to do more hyperparameter tuning here, but that's probably another PR.

Reply:
Haven't thought about hp tuning at all. Plan is to think about that after there's most of the pieces of the AIRL/GAIL pipelines available.

Adding `n_epochs = 2000` to the new configs.